CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+## vNext
```

Comment:
This should be in the 10.5.0 section - it is an upcoming unreleased version, we just gave it a name to be explicit that it's different from the v10 releases.